tests/vmi_multus_test.go

```diff
@@ -885,6 +872,56 @@ var _ = Describe("[Serial]SRIOV", func() {
 				return libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))
 			}, 15*time.Second, time.Second).Should(Succeed())
 		})
+
+		Context("With VLAN", func() {
+			var (
+				macVlaned1   string
+				macVlaned2   string
+				macNonVlaned string
```

Comment:
I would remove the `Vlaned` `NonVlaned` postfix, the VM context is implied by the CIDR variable names 
(878-880 line and 898-900)

Reply:
Done.